bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLoggerAllocator.java

```diff
@@ -185,7 +185,7 @@ private synchronized BufferedLogChannel allocateNewLog(File dirForNextEntryLog,
 
 
     private synchronized void closePreAllocateLog() {
-        if (preallocatedLogId != -1) {
+        if (preallocation != null) {
```

Comment:
this change looks unrelated

Reply:
Yeah, that was causing a test to fail so I fixed it.